src/Raven.Server/Documents/Sharding/Handlers/Processors/Replication/ShardedReplicationHandlerProcessorForGetTombstones.cs

```diff
@@ -51,6 +51,11 @@ public GetTombstonesPreviewResult Combine(Memory<GetTombstonesPreviewResult> res
             {
                 var final = new GetTombstonesPreviewResult();
 
+                foreach (var result in results.Span)
+                {
+                    result.Tombstones.Sort(TombstonesPreviewComparer.Instance);
+                }
```

Comment:
I think this is wrong.
1. will this materialize all tombstones?
2. would the continuation token work properly? you mess the original order so the `skip` will be wrong

Reply:
You are right. It's same as replication conflicts. Done.